script.js

```diff
@@ -1,4 +1,217 @@
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+// Mode switch between two players.
+const PLAYER_1 = "player 1";
+const PLAYER_2 = "player 2";
+const DISPLAY_DICE_MODE = "display dice";
+const CHOOSE_DICE_COMBI = "choose dice combination";
+const COMPARE_RESULT_MODE = "check who win";
+const HIGHEST_COMBI_MODE = "high";
+const LOWEST_COMBI_MODE = "low";
+let gameMode = DISPLAY_DICE_MODE;
+let whosTurn = PLAYER_1;
+// temporary holder for the dice rolled each turn.
+let tempDiceHolder = [];
+//Container for the number combination players chose.
+let player1DiceCombi = [];
+let player2DiceCombi = [];
+// Keep total score for each player.
+let player1TotalScore = 0;
+let player2TotalScore = 0;
+
+// helper function
+const diceRoll = function () {
+  // round up to the next largest integer
+  let randomDiceNumber = Math.ceil(Math.random() * 6);
+  return randomDiceNumber;
+};
+
+const displayDice = function () {
+  let dice1 = diceRoll();
+  let dice2 = diceRoll();
+  let message = "";
+  tempDiceHolder = [];
+  tempDiceHolder.push(dice1, dice2);
+  message = `You rolled ${dice1} and ${dice2}`;
+  console.log("dice 1");
+  console.log(dice1);
+  console.log("dice 2");
+  console.log(dice2);
+  console.log("game mode");
+  console.log(gameMode);
```

Comment:
it is good to use console log during development but remove them during code submission for clarity and readability of code